packages/forms/src/Form.js

```diff
@@ -132,6 +132,9 @@ class Form extends React.Component {
 			}
 			return <UIForm {...props} />;
 		}
+		if (Array.isArray(this.props.data.uiSchema)) {
```

Comment:
Not a big deal, but isn't that the most determinant condition ? If so it should be the first test in the `render` method.
1. if it's uiSpec v1 --> render directly the UIForm with all the props
2. if we have the uiForm flag, it means that we have uiSpec v0 but we want to use UIForm --> widget adaptation and render UIForm
3. else render RJSFForm